src/query/typed_value.cpp

```diff
@@ -215,6 +216,9 @@ TypedValue::TypedValue(const TypedValue &other, utils::MemoryResource *memory) :
     case Type::Duration:
       new (&duration_v) utils::Duration(other.duration_v);
       return;
+    case Type::Function:
+      LOG_FATAL("Unsupported TypedValue::Function copy ctor");
```

Comment:
Why you couldn't copy `std::function`?

Reply:
I just disabled it not to copy it unnecessarily. I will enable it, makes sense